wordpress-mobile/WordPress-iOS
# Zendesk: always include source and platform tags.

Fixes #n/a

This fixes an issue where source and platform tags were not added to a Zendesk ticket if the account has no blogs. This was most noticeable during the auth process since the user was not logged in yet.

Previously, when creating the tags array, the check for blog count came first and would immediately return if there were no blogs. Now, the source and platform tags are added before the blog count is checked.

To test:

- Log out of the app.
- Select log in.
- On any screen in the auth process, select `Help` in the nav bar.
- Select `Contact Support` (you may need to enter your name & email before proceeding).
  - Verify `source` is now included in the logged event: 
  `🔵 Tracked: support_new_request_viewed <source: origin:login-wpcom-password>`
- Submit a request.
- Find your ticket in Zendesk.
  - Verify it has an `origin:` tag and it matches was was logged out above.


| Before | After |
|--------|-------|
| <img width="281" alt="Screen Shot 2021-08-25 at 12 00 36 PM" src="https://user-images.githubusercontent.com/1816888/130849913-503871aa-6696-453b-9c48-1e99624813e8.png"> | <img width="280" alt="Screen Shot 2021-08-25 at 12 01 02 PM" src="https://user-images.githubusercontent.com/1816888/130849937-f8581c15-6772-4d1d-854b-19a347186e84.png"> |

## Regression Notes
1. Potential unintended areas of impact


2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
